### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

* Rephrase "on-premise" to "local development"
* Move maintenance mode docs to drafts

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _data/sidebars/cloud_dev_sidebar.yml [hunk 1]
```diff
@@ -71,8 +71,8 @@ entries:
             url: /docs/cloud/dev/spryker-cloud-commerce-os/configure-deployment-pipelines/configuring-bitbucket-pipelines.html
       - title: Configuring data import from an S3 bucket
         url: /docs/cloud/dev/spryker-cloud-commerce-os/configuring-data-import-from-an-s3-bucket.html
-      - title: Enable and disable maintenance mode
-        url: /docs/cloud/dev/spryker-cloud-commerce-os/enable-and-disable-maintenance-mode.html
+#      - title: Enable and disable maintenance mode
+#        url: /docs/cloud/dev/spryker-cloud-commerce-os/enable-and-disable-maintenance-mode.html
```
Comment (VadymSachenko) [inline _data/sidebars/cloud_dev_sidebar.yml:75]: What you use `#` for?

File: docs/scos/dev/setup/installing-spryker-with-docker/installing-spryker-with-docker.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Before you install Spryker with Docker, review the installation prerequisites fo
 
 ## Installation
 
-There are several ways to configure a Docker-based instance of Spryker. See [Choosing an installation mode](/docs/scos/dev/setup/installing-spryker-with-docker/installation-guides/choosing-an-installation-mode.html) to learn about them in detail.
+There are several ways to install a Docker-based instance of Spryker. See [Choosing an installation mode](/docs/scos/dev/setup/installing-spryker-with-docker/installation-guides/choosing-an-installation-mode.html) to learn about them in detail.
```
Comment (VadymSachenko) [inline docs/scos/dev/setup/installing-spryker-with-docker/installing-spryker-with-docker.md:51]:
```suggestion
There are several ways to install a Docker-based instance of Spryker. For more details about these ways, see [Choosing an installation mode](/docs/scos/dev/setup/installing-spryker-with-docker/installation-guides/choosing-an-installation-mode.html).
```

File: docs/scos/dev/setup/setup.md [hunk 3]
```diff
@@ -6,10 +6,10 @@ redirect_from:
...
 ---
 
-This section contains instructions for installing Spryker Commerce OS on premises.
+This section contains instructions for installing local instances of Spryker Commerce OS. Local instances are used for development, demos, or experimentation purposes. To launch a live shop on [Spryker PaaS](https://spryker.com/platform-page-commerce/), [contact us](https://spryker.com/contact-us-commerce/).
 
-You can install on-premises Spryker with Docker or Development Virtual Machine. We recommend installing Spryker with Docker. For installation instructions, see one of the following documents:
+To replicate a PaaS environment on your machine, see [Installing Spryker with Docker](/docs/scos/dev/setup/installing-spryker-with-docker/installing-spryker-with-docker.html).
 
-* [Installing Spryker with Docker](/docs/scos/dev/setup/installing-spryker-with-docker/installing-spryker-with-docker.html)
+For other solutions that are not compatible with PaaS environments, see the following:
```
Comment (VadymSachenko) [inline docs/scos/dev/setup/setup.md:13]:
```suggestion
For other solutions that are not compatible with PaaS environments, see the following documents:
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]

File: _data/sidebars/cloud_dev_sidebar.yml (hunk 1 above)
Comment (andriitserkovnyi) [reply _data/sidebars/cloud_dev_sidebar.yml:75]: To comment out this entry of the sidebar

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**